docs/UserGuide.md

```diff
@@ -63,17 +64,24 @@ Prints out a message on how to use ATHENA.
 
 
 ### Add a task: `add` or `a`
-Adds a task to the planner.
+This command adds a task to the planner.
 
-#### Format
-`add n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
-`a n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+The guide for this command is split into three sections:
+
+* [Basic Usage](#basic-usage) (For users new to command line applications)
+* [Intermediate Usage](#intermediate-usage) (For users comfortable with command line applications)
+* [Advanced Usage](#advanced-usage) (Extension to *Intermediate Usage*)
+
+#### Basic Usage
+
+The `add` command accepts 7 parameters.
 
-#### Parameters
 * `NAME` is the name of the task. It is a compulsory parameter the user has to input.
+
+The following parameters are optional. If they are left empty, ATHENA will use the default value assigned.
 * `TIME` is the time to start doing this task (**HHMM**). For example, 1100.
 
-   Default: Allocated by Athena (only available for non recurring tasks)
+   Default: Allocated by Athena (only available for non-recurring tasks)
```

Comment:
I think we shouldn't mention default values in basic usage? Later CELC ppl get confused.